addons/dialogue_nodes/nodes/sub_nodes/ForkOption.tscn

```diff
@@ -0,0 +1,74 @@
+[gd_scene load_steps=4 format=3 uid="uid://nfp4t2r7wfhi"]
+
+[ext_resource type="Script" path="res://addons/dialogue_nodes/nodes/sub_nodes/forkOption.gd" id="1_bqi8r"]
+[ext_resource type="Texture2D" uid="uid://b10pucyepsb2s" path="res://addons/dialogue_nodes/icons/Filter.svg" id="2_4xusw"]
+[ext_resource type="Texture2D" uid="uid://cf5ef41yo6jl6" path="res://addons/dialogue_nodes/icons/Reload.svg" id="3_pf2k3"]
+
+[node name="ForkOption1" type="HBoxContainer" groups=["fork_node_options"]]
```

Comment:
Maybe remove the group since its not used anywhere.

Reply:
Group deleted.